generators/entity-client/templates/ionic/e2e/entities/_entity.e2e-spec.ts.ejs

```diff
@@ -0,0 +1,210 @@
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-%>
+import { browser, by, element, ExpectedConditions as ec } from 'protractor';
+import { LoginPage } from '../../pages/login.po';
+import {
+  <%= entityClass %>ComponentsPage,
+  <%_ if (!readOnly) { _%>
+      <%= entityClass %>DetailPage,
+      <%= entityClass %>UpdatePage
+  <%_ } _%>
+} from './<%= entityFileName %>.po';
+
+
+describe('<%= entityClass %> e2e test', () => {
+
+  let loginPage: LoginPage;
+  let <%= entityInstance %>ComponentsPage: <%= entityClass %>ComponentsPage;
+  <%_ if (!readOnly) { _%>
+  let <%= entityInstance %>UpdatePage: <%= entityClass %>UpdatePage;
+  let <%= entityInstance %>DetailPage: <%= entityClass %>DetailPage;
+  <%_ } _%>
+  let initNumberOfEntities: number;
+  const COMPONENT_TITLE = '<%= entityClassPlural%>';
+  const SUBCOMPONENT_TITLE = '<%= entityClass %>';
+
+  <%_ fields.forEach((field) => {
+    const fieldName = field.fieldName;
+    const fieldNameCapitalized = field.fieldNameCapitalized;
+    const fieldType = field.fieldType;
+    const fieldTypeBlobContent = field.fieldTypeBlobContent;
+    const fieldIsEnum = field.fieldIsEnum;
+    const fieldValidateSampleString = field.fieldValidateSampleString;
+    const minLength = field.fieldValidateRulesMinlength;
+_%>
+<%_ if (['Integer', 'Long', 'Float', 'Double', 'BigDecimal'].includes(fieldType)) { _%>
+  const <%= fieldName %> = '5';
+<%_ } else if (fieldType === 'LocalDate') { _%>
+<%_ } else if (['Instant', 'ZonedDateTime'].includes(fieldType)) { _%>
+<%_ } else if (fieldType === 'Duration') { _%>
+  const <%= fieldName %> = 'PT12S';
+<%_ } else if (['byte[]', 'ByteBuffer'].includes(fieldType) && fieldTypeBlobContent === 'text') { _%>
+  const <%= fieldName %> = '<%= fieldName %>';
+<%_ } else if (['byte[]', 'ByteBuffer'].includes(fieldType)) { _%>
+  <%# nothing field to upload byte image -%>
+<%_ } else if (fieldIsEnum) { _%>
+  <%# it will be set last choose -%>
+<%_ } else if (fieldType === 'UUID') { _%>
+  const <%= fieldName %> ='64c99148-3908-465d-8c4a-e510e3ade974';
+<%_ } else if (fieldType === 'String' && fieldValidateSampleString) { _%>
+  const <%= fieldName %> = '<%= fieldValidateSampleString %>';
+<%_ } else if (fieldType === 'String' && fieldName.length < minLength) { _%>
+  <%_ let value = 'A'.repeat(minLength - fieldName.length );
+   value = fieldName + value; _%>
+  const <%= fieldName %> =  '<%= value_%>';
+<%_ } else if (fieldType !== 'Boolean') { _%>
+  const <%= fieldName %> = '<%= fieldName %>';
+<%_ } _%>
+<%_ }); _%>
+
+  beforeAll(async () => {
+    loginPage = new LoginPage();
+    await loginPage.navigateTo('/');
+    await loginPage.signInButton.click();
+    <%_ if (authenticationType !== 'oauth2') { _%>
+    await browser.wait(ec.elementToBeClickable(loginPage.loginButton), 3000);
+    <%_ } _%>
+    await loginPage.login('admin', 'admin'); // use process.env.E2E_USERNAME if you want to use env variables
```

Comment:
We might want to make this similar to what I have pending for JHipster. https://github.com/jhipster/generator-jhipster/pull/11448

```ts
const username = process.env.E2E_USERNAME || 'admin';
const password = process.env.E2E_PASSWORD || 'admin';
```